CONTRIBUTING.md

```diff
@@ -87,77 +87,145 @@ You have done! Now it is possible to insert breakpoints and debug server and cli
 
 ## Setup additional components in development environment
 
-### Automatic annotation
--   Install OpenVINO on your host machine according to instructions from
-[OpenVINO website](https://docs.openvinotoolkit.org/latest/index.html)
+### DL models as serverless functions
 
--   Add some environment variables (copy code below to the end of ``.env/bin/activate`` file):
-```sh
-    source /opt/intel/openvino/bin/setupvars.sh
+Install [nuclio platform](https://github.com/nuclio/nuclio)
 
-    export OPENVINO_TOOLKIT="yes"
-    export IE_PLUGINS_PATH="/opt/intel/openvino/deployment_tools/inference_engine/lib/intel64"
-    export OpenCV_DIR="/usr/local/lib/cmake/opencv4"
-    export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/intel/openvino/inference_engine/lib/intel64"
+- You have to install `nuctl` command line tool to build and deploy serverless
+functions. Download [the latest release](https://github.com/nuclio/nuclio/releases).
```

Comment:
Need more specific instructions how to install. Not everyone has good experience with ubuntu. 
For example I did follow steps with the downloaded file:

```bash
chmod +x nuctl-1.4.8-linux-amd64
mv nuctl-1.4.8-linux-amd64 nuctl
sudo mv nuctl /usr/bin
```

Reply:
@bsekachev guess it will be better to add link to official documentation like this (<https://github.com/nuclio/nuclio/blob/development/docs/reference/nuctl/nuctl.md#download>)